tests/Bundle/Config/ConfigResolverTest.php

```diff
@@ -51,13 +51,24 @@ public function testDefinesAFluentInterface(): void
      */
     public function testAddsTheBundleConfigs(array $configs, array $expectedResult): void
     {
+        // Shuffle the input around to ensure the input order does not alter the output.
+        // This does not guarantee failure if the random input already provides the expected output but is better
+        // than nothing - at least we have a higher probability than 0 to catch these issues here.
+        shuffle($configs);
```

Comment:
```suggestion
        mt_srand(0);
        shuffle($configs);
        mt_srand();
```

Would make this test reproducible too. 
Or maybe run the test five times with different seeded random orders?

Reply:
Either sounds good to me.

Reply:
Running it five times with `mt_srand(0)` til `mt_srand(4)` would be my preferred solution.